backend/routes/db.py

```diff
@@ -343,16 +275,164 @@ async def get_concepts_post_route(request: Request, id: Union[List[str], None] =
     return await get_concepts_route(request, id=id, table=table)
 
 
+sessions_cache: Dict = {}
```

Comment:
```suggestion
SESSIONS_CACHE: Dict = {}
```
IDK if this is plural by accident (other variables seem to be singular), but at the moment I have no objections.